detox/src/utils/pipeCommands.test.js

```diff
@@ -0,0 +1,102 @@
+const pipeUtils = require('./pipeCommands');
+
+describe('pipeUtils', () => {
+  describe('escape in quoted string', () => {
+    describe('on win32', () => {
+      const escape = pipeUtils.escape.inQuotedString.win32;
+
+      it('should escape double quotes with double quotes', () => {
+        expect(escape('Says "Hello"')).toBe('Says ""Hello""');
+      });
+    });
+
+    describe('on *nix', () => {
+      const escape = pipeUtils.escape.inQuotedString.nix;
+
+      it('should escape double quotes with slash', () => {
+        expect(escape('Says "Hello"')).toBe('Says \\"Hello\\"');
+      });
+    });
+  });
+
+  describe('escape in quoted regexp', () => {
+    describe('on win32', () => {
+      const escape = pipeUtils.escape.inQuotedRegexp.win32;
+
+      it('should escape only double quotes', () => {
+        expect(escape('^Hello\\s ".*"$')).toBe('^Hello\\s "".*""$');
+      });
+    });
+
+    describe('on *nix', () => {
+      const escape = pipeUtils.escape.inQuotedRegexp.nix;
+
+      it('should not escape non-special characters', () => {
+        expect(escape('bundle_name')).toBe('bundle_name');
+      });
+
+      it('should escape double quotes with slash', () => {
+        expect(escape('"a')).toBe('\\"a');
+      });
+
+      it('should escape [\\]', () => {
+        const bundleId = '[kworker\\0:0]';
+        expect(escape(bundleId)).toBe('\\[kworker\\\\0:0\\]');
+      });
+
+      it('should escape ^*$', () => {
+        const bundleId = '^ma*tch$';
+        expect(escape(bundleId)).toBe('\\^ma\\*tch\\$');
+      });
+
+      it('should escape dots', () => {
+        const bundleId = 'com.company.bundle';
+        expect(escape(bundleId)).toBe('com\\.company\\.bundle');
+      });
+    });
+  });
+
+  describe('search by regexp pipe command', () => {
+    const regexp = '^Hello\\s ".*"$';
+
+    describe('on win32', () => {
```

Comment:
These tests seem to test nothing. we already know what `pipeUtils.search.regexp.win32` is. There's probably no need to rewrite the same logic in the test.